main-site/components/meet-the-team/team-column/TeamColumn.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { TeamColumnProps } from "../../../../shared-ui/lib/types";
+import { StyledTeamColumn, StyledLabel, StyledHeadshot } from "./TeamColumn.styles";
+
+const TeamColumn: React.FC<TeamColumnProps> = ({columnInfo}) => {
+  const listOfPictures: string[] = columnInfo.listOfPictures;
+    return (
+      <StyledTeamColumn>
+          <StyledLabel>
+              {columnInfo.teamLabel}
+          </StyledLabel>
```

Comment:
For the StyledLabel, when its 2 lines, it shifts all the images in the column down. 
<img width="871" alt="Screen Shot 2022-10-18 at 3 20 03 PM" src="https://user-images.githubusercontent.com/59738880/196524061-3e766e15-dc63-4d6d-947c-d122788da1a9.png">

Can we always keep this as two lines by setting the width or max-width and then add margin: -0.4em  ? This will shift the images below it up so it can match the other columns

You can pass in a prop to your styled label that represents if ur thing is two lines or not and then apply the margin according to that